source/Preferences.h

```diff
@@ -28,6 +28,12 @@ class Preferences {
 		adaptive,
 	};
 
+	enum class DateFormat : int {
+		dmy = 0,
+		mdy,
+		ymd
+	};
```

Comment:
Optional: I think you can add comments to each enum value and IDEs will display that context on hovering the value.

Reply:
Okay I'll add comments, with example dates for reference.

Reply:
25d4f5b4614303c91ad6ee6bae02be74052442c4